src/benchmarks/micro/libraries/System.IO.FileSystem/Perf.FileStream.cs

```diff
@@ -2,152 +2,299 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
 using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
 using MicroBenchmarks;
 
 namespace System.IO.Tests
 {
     [BenchmarkCategory(Categories.Libraries)]
     public class Perf_FileStream
     {
-        private const int DefaultBufferSize = 4096;
-        
-        [Params(DefaultBufferSize / 8, 200000)]
-        public int BufferSize;
+        private const int OneKiloByte = 1_000;
+        private const int HalfKiloByte = OneKiloByte / 2;
+        private const int FourKiloBytes = OneKiloByte * 4; // default Stream buffer size
+        private const int OneMegaByte = OneKiloByte * 1_000;
+        private const int HundredMegaBytes = 1_000_000 * 100;
```

Comment:
should these all be powers of two? Ie., what you call Kibibytes, Mibibytes etc. 

Since the default FileStream buffer is a Kibibyte = 1024, not 1000.